src/demo/app/fancy-form/fancy-field-demo.component.ts

```diff
@@ -0,0 +1,264 @@
+import { Component } from '@angular/core';
+import { DemoModel } from '../model/demo.model';
+import { HalDoc } from 'ngx-prx-styleguide';
+
+@Component({
+  moduleId: module.id,
+  selector: 'fancy-field-demo',
+  template: `
+    <section class="main demo">
+      <h1>FancyField</h1>
+      <p>
+        The Fancy Field component is used to create PRX forms fields. It supports, text, number, checkbox, select, and
+        textarea input types. It will also handle formatting of labels and hints along with your projected content. This
+        component is intended to work with models extended from BaseModel, but it will also create disabled form fields
+        without a model. When working with BaseModel, any validation errors that occur will be shown within the fancy field.
+      </p>
+      <ul>
+        <li>
+          <code>@Input() model: BaseModel</code>
+           - the model containing the named property
+        </li>
+        <li>
+          <code>@Output() onChange: EventEmitter&lt;any&gt;</code>
+           - whenever the field changes, the new value is emitted
+        </li>  
+        <li>
+          <code>@Input() name: string</code>
+           - name of model attribute, and optional explicit changed/invalid bindings
+        </li>
+        <li>
+          <code>@Input() changed: string</code>
+           - an alternate field name can be provided for BaseModel field change checking, otherwise <code>name</code> is used
+        </li>
+        <li>
+          <code>@Input() invalid: string</code>
+           - an alternate field name can be provided for BaseModel field validation, otherwise <code>name</code> is used
+        </li>
+        <li>
+          <code>@Input() label: string</code>
+           - the field label presented to the user
+        </li>
+        <li>
+          <code>@Input() invalidlabel: string</code>
+           - an alternate label can be provided for validation messages, otherwise <code>label</code> is used
+        </li>
+        <li>
+          <code>@Input() hideinvalid: boolean</code>
+           - set to false if validation messages are not desired
+        </li>
+        <li>
+          <code>@Input() advancedConfirm: string</code>
+           - a confirmation message for "advanced" fields
+        </li>
+        <li>
+          <code>@Input() strict: boolean</code>
```

Comment:
could be worth explaining meaning of strict? (strict:true will prevent save if invalid, right?)